mobile-browser-based-version/src/components/building_frames/ImageTestingFrame.vue

```diff
@@ -326,17 +324,12 @@ export default {
     },
     downloadPredictionsCsv(){
       console.log(this.predictions)
-      let pred = ""
-      let header_length = 0
-      for (const [id, prediction] of Object.entries(this.predictions)){
-        header_length = prediction.length
-        pred += id + ","+ prediction.map(dict => dict["className"]+","+dict["probability"]).join(',')+"\n"
```

Comment:
I might have deleted a part you've updated @MarcelTorne. Could you confirm that the new version is correct? Thanks!

Reply:
Resolved: brought back the original file in from the maste branch.